source/manual/manage-ruby-dependencies.html.md

```diff
@@ -32,9 +35,9 @@ Go to [Dependabot admin][admin] and click "Bump now" for your project
 
 There are 2 safeguards to prevent unauthorised code changes. Firstly, Dependabot can only update the repositories that we [explicitly allow on GitHub][access]. This prevents code changes to other repos. Secondly, we've [set up branch protection](/manual/configure-github-repo.html#auto-configuration) for all repos with the `govuk` label. This prevents Dependabot from writing directly to master.
 
-[ext]: https://github.com/alphagov/govuk-rfcs/blob/master/rfc-052-pull-request-merging-process.md#a-change-from-an-external-contributor
-[tech-debt]: https://trello.com/c/tBEHwkI9/62-wip-out-of-date-dependencies
-[access]: https://github.com/organizations/alphagov/settings/installations/64726
+[ext]: https://docs.publishing.service.gov.uk/manual/merge-pr.html
+[tech-debt]: https://trello.com/c/tBEHwkI9
```

Comment:
This takes me to an archived Trello card, but it is the right board.

Reply:
Thanks, fixed.